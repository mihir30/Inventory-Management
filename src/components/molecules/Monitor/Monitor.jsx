import React from 'react';

const Monitor = () => {

    return (
            <div className='group2'>
                <label className='form-label'>
                    Size,inch
                </label>
                <input className='form-control' type='number' defaultValue={23}/>
            </div>
    )
}

export default Monitor;