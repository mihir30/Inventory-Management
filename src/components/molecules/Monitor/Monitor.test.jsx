import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import ReactDom from "react-dom";
import Monitor from './Monitor';

describe("Monitor Component", () => {
    it('renders the component properly', () => {
        const divEl = document.createElement("div");
        ReactDom.render(<BrowserRouter><Monitor></Monitor></BrowserRouter>, divEl); 
        
    });


    it('should have the right monitor component snapshot', () => {
        const tree = renderer.create().toJSON();
        expect(tree).toMatchSnapshot();
    })

});