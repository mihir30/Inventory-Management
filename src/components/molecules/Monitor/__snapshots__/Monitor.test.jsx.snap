// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Monitor Component should have the right monitor component snapshot 1`] = `null`;
